File: problemas/1100/1133.dart
import 'dart:io';
import 'dart:math';

void main() {
  final x = int.parse(stdin.readLineSync());
  final y = int.parse(stdin.readLineSync());
  final menor = min(x, y);
  final maior = max(x, y);

  for (int i = menor + 1; i < maior; i++) {
    if (i % 5 == 2 || i % 5 == 3) {
      stdout.writeln(i);
    }
  }
}

File: problemas/1100/1133.js
const [x, y] = require('fs').readFileSync('/dev/stdin', 'utf8').split('\n');
const [menor, maior] = [Math.min(x, y), Math.max(x, y)];

for (let i = menor + 1; i < maior; i++) {
  if (i % 5 == 2 || i % 5 == 3) {
    console.log(i);
  }
}